Plugin/Quote/Model/AbstractCheckoutTrigger.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Yotpo\SmsBump\Plugin\Quote\Model;
+
+use Yotpo\SmsBump\Model\Sync\Checkout\Processor as CheckoutProcessor;
+use Magento\Framework\Registry;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Yotpo\SmsBump\Model\Config;
+use Magento\Quote\Model\Quote as QuoteModel;
```

Comment:
I think we should remove the ModelQuote reference, and keep it as Quote, as it a bit confusing. What do you think?

Reply:
It would require changing the class name. it would work either way.

Reply:
@DanielxC Note that the namespace is also `Quote\Model`, so I do agree that there should be some diferentiation.

Reply:
We are using it as Quote in other places:
[**1**](https://github.com/YotpoLtd/magento2-module-messaging/blob/master/Model/AbandonedCart/Data.php#L6)
[**2**](https://github.com/YotpoLtd/magento2-module-messaging/blob/master/Plugin/Checkout/DefaultConfigProviderPlugin.php#L14)
So I would keep it as Quote, but up to you

Reply:
But because we extended the Quote model with DI, let's keep it QuoteModel.